nextjs/context/AuthContext.tsx

```diff
@@ -58,6 +59,7 @@ export function AuthProvider({ children }: AuthProviderProps) {
     // If no error and we have user data, return it
     if (res.status === 200 && data) {
       const token = data.access_token
+      localStorage.setItem('userToken', token)
```

Comment:
Is `userToken` too generic or vague nomenclature?

Reply:
The userToken is good in this case I believe since it provides all the user's and session's information when it is parsed.